typings/index.d.ts

```diff
@@ -2800,9 +2889,9 @@ export type ApplicationFlagsString =
   | 'EMBEDDED';
 
 export interface AuditLogChange {
-  key: string;
-  old?: unknown;
-  new?: unknown;
+  key: APIAuditLogChange['key'];
+  old?: APIAuditLogChange['old_value'];
+  new?: APIAuditLogChange['new_value'];
 }
```

Comment:
Does this... pick up the unions? If it does, that's awesome!

Reply:
It does! It's quite awesome
![image](https://user-images.githubusercontent.com/8060149/127725284-6c313bfc-fdb7-4bcc-9c68-e15e93f19fcb.png)
![image](https://user-images.githubusercontent.com/8060149/127725289-b719dac2-6c3e-404c-8b8b-d8dbe8801f2d.png)